content/blog/entries/what-is-up-ccos/contents.lr

```diff
@@ -0,0 +1,35 @@
+title: What is up? - CCOS Revamp
+---
+categories:
+outreachy
+tech
+open-source
+---
+author: dhruvi16
+---
+series: outreachy-2020-CCOS
+---
+pub_date: 2020-07-20
+---
+body:
+
+
+Well in my previous blog, I demonstrated what was my Outreachy project about. Here I will talk about my progress in the past 7 weeks.
```

Comment:
I would re-phrase "Well in my previous blog, I demonstrated what was my Outreachy project about." to "In my previous blog, I demonstrated what my Outreachy project was about."